cou/cli.py

```diff
@@ -143,13 +160,30 @@ async def run_upgrade(
     :type model_name: Optional[str]
     :param backup_database: Whether to create database backup before upgrade.
     :type backup_database: bool
-    :param interactive: Whether to run upgrade interactively.
-    :type interactive: bool
+    :param prompt: Whether to prompt to run upgrade interactively.
+    :type prompt: bool
     :param quiet: Whether to run upgrade in quiet mode.
     :type quiet: bool
     """
     analysis_result, upgrade_plan = await analyze_and_plan(model_name, backup_database)
-    logger.info(upgrade_plan)
+    logger.debug(upgrade_plan)
```

Comment:
**question (non-blocking):** 

Why replace `logger.info` with `logger.debug` and `print`?

Reply:
Because we will always show upgrade_plan. Reducing logging level means we will not double-print the plan unless the user run `cou upgrade -vvv/-vvvv`. Btw, this change is part of #196, where this PR is based.